### PR TITLE
Fixed 500 error page

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Fixed 500 error page by removing nav and footer and use of {{ asset server url }}

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [500 page](http://0.0.0.0:8001/500)
- see that the page looks good

## Issue / Card

Fixes # 2591

## Screenshots
image

### CODE REVIEW

**Comment by spencerbygraves** [comment]
@pmahnke does it need the padding-top on large screens?

File: templates/500.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{% extends "templates/one-column.html" %}
+{% extends "templates/one-column_no_nav.html" %}
```
Comment (anthonydillon) [inline templates/500.html]: Why did you remove the navigation on the 500 page? This is get the user stuck there. Just because the current page is 500'ing it doesn't mean other pages wont work as expected.

Comment (pmahnke) [reply templates/500.html]: @anthonydillon oddly, it wasn't executing, and showing empty navigation - and it didn't parse the {{asset_server}} variable - see https://github.com/canonical-websites/www.ubuntu.com/issues/2591

Comment (anthonydillon) [reply templates/500.html]: Makes sense, dynamic (YAML) navigation will not work in a server error situation. If we hard code it, it will get out of date. So the Ubuntu logo is ok for now.

**Comment by pmahnke** [comment]
@spencerbygraves should be ok for a re-review - http://www.ubuntu.com-pr-2592.run.demo.haus/500

**Comment by pmahnke** [comment]
@spencerbygraves you are right, I have updated it

**Comment by anthonydillon** [comment]
:+1:
